src/bdk.udl

```diff
@@ -283,6 +283,12 @@ interface DescriptorSecretKey {
   [Throws=BdkError]
   constructor(Network network, string mnemonic, string? password);
 
+  [Name=generate, Throws=BdkError]
```

Comment:
You should be able to remove the `Throws=BdkError` for both of these new constructors and update code to just return `Self`.

Reply:
Done!